bigchaindb/commands/bigchaindb.py

```diff
@@ -95,21 +95,35 @@ def run_configure(args):
 
 @configure_bigchaindb
 def run_upsert_validator(args):
-    """Store validators which should be synced with Tendermint"""
-
-    b = bigchaindb.BigchainDB()
-    public_key = public_key_from_base64(args.public_key)
-    validator = {'pub_key': {'type': 'ed25519',
-                             'data': public_key},
-                 'power': args.power}
-    validator_update = {'validator': validator,
-                        'update_id': VALIDATOR_UPDATE_ID}
-    try:
-        query.store_validator_update(b.connection, validator_update)
-    except MultipleValidatorOperationError:
-        logger.error('A validator update is pending to be applied. '
-                     'Please re-try after the current update has '
-                     'been processed.')
+    """Initiate and manage elections to change the validator set"""
+
+    b = BigchainDB()
+
+    # Call the function specified by args.action, as defined above
+    globals()[f'run_upsert_validator_{args.action}'](args, b)
```

Comment:
Will there be any other action? Can you also add a docs string for what the args can be?

Reply:
Yes. There is also going to be a `run_upsert_validator_show`, `_approve`, and `_status`, as detailed in BEP-21.

Added a docstring detailing the args